Packs/GoogleCloudFunctions/Integrations/GoogleCloudFunctions/GoogleCloudFunctions.py

```diff
@@ -169,15 +171,38 @@ def format_parameters(parameters: str) -> str:
     return json.dumps(dict(pairs))
 
 
+def set_default_region(region):
+    # when region is empty, set it to '-' meaning all regions
+    # note : demisto.params().get('region','-') did not worked on Demisto
+    if region is None:
```

Comment:
Handle also the case of empty string:
```suggestion
    if not region:
```

Reply:
Done